numpydoc/docscrape.py

```diff
@@ -555,7 +555,16 @@ def __str__(self):
 
 class ClassDoc(NumpyDocString):
 
-    extra_public_methods = ['__call__']
+    extra_public_methods = ['_call',
+                            '_apply',
+                            '_lincomb',
+                            '_multiply',
+                            '_divide',
+                            '_dist',
+                            '_norm',
+                            '_inner']
+
+    excluded_methods = ['next']
```

Comment:
obsolete?

Reply:
Nope, I think this is everywhere?

Reply:
Which of our classes has a `next` method?

Reply:
In python 2, basically all of them, e.g.

```
>>> rn = odl.rn(3)
>>> rn.next
<bound method NumpyTensorSpace.next of rn(3)>
```

Reply:
Oh that's from `future`'s `object` class. Okay, then we need to keep it.